### PR TITLE
Feature/get balance stream

### LINKED ISSUE #103: Get balance stream
I'd like to get a stream of balance for an address. DRC has that [functionality](https://github.com/trufflesuite/drizzle-react-components/blob/develop/src/AccountData.js).

The closest thing DU has is `contract-state-stream`, but that's for contracts only and comes with a lot of other info you might not need.

### PR DESCRIPTION
**Pull request by honestbonsai**
Resolves #103

### CODE REVIEW

File: packages/get-balance-stream/test-app/truffle-config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const path = require("path");
+
+module.exports = {
+  // See <http://truffleframework.com/docs/advanced/configuration>
+  // to customize your Truffle configuration!
+  contracts_build_directory: path.join(__dirname, "client/src/contracts")
```
Comment (adrianmcli) [inline packages/get-balance-stream/test-app/truffle-config.js]: Man, I'd really like to use nano-react-app instead so we don't have to do this funky thing.

**Comment by adrianmcli** [comment]
@honestbonsai Code looks good to me, can you rebase off master and make sure tests run properly?

File: packages/get-balance-stream/test-app/truffle-config.js (hunk 1 above)
Comment (honestbonsai) [reply packages/get-balance-stream/test-app/truffle-config.js]: I think that should be part of another PR to refactor everything.

**Comment by honestbonsai** [comment]
@adrianmcli I've rebased this PR off `master` and its tests pass.

`contract-state-stream` tests fail in `master` and this PR, so I'd say that's an issue for another PR.

**Comment by adrianmcli** [comment]
Thanks, can you open up another PR for that?

**Pull request merged**